commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -80,7 +80,7 @@ cpdef run( filename_trk, path_out, filename_peaks = None, filename_mask = None,
     # fiber-tracts from .trk
     print '\t\t* tractogram'
     try :
-        fib, trk_hdr = nibabel.trackvis.read( filename_trk )
+        _, trk_hdr = nibabel.trackvis.read( filename_trk, as_generator=True )
```

Comment:
In `nibabel.trackvis.read()` the default parameter `as_generator` is `False` and the trucks are returned as sequence. If we explicitly use `as_generator=True`the trucks are returned as generator.